docs/approach.rst

```diff
@@ -149,14 +178,30 @@ of the other echoes (which it is).
     We do, however, make it accessible as an alternative combination method
     in the t2smap workflow.
 
+Denoising
+`````````
+The next step is an attempt to remove noise from the data. 
+This process can be 
+broadly seperated into three steps: **decomposition, metric calculation** and 
+**component selection**. 
+Decomposition reduces the dimensionality of the 
+optimally combined data using PCA and then an ICA. 
```

Comment:
```suggestion
optimally combined data using `Principal Components Analysis (PCA)`_ and then an `Independent Components Analysis (ICA)`_. 
```

The links I'd suggest using are: 
- PCA: https://scikit-learn.org/stable/modules/generated/sklearn.decomposition.PCA.html
- ICA: https://scikit-learn.org/stable/modules/generated/sklearn.decomposition.FastICA.html

Since this should be what we're using in-code !